pkg/datapath/maps/map.go

```diff
@@ -165,6 +165,10 @@ func (ms *MapSweeper) RemoveDisabledMaps() {
 		maps = append(maps, "cilium_ipv4_frag_datagrams")
 	}
 
+	if !option.Config.EnableBandwidthManager {
+		maps = append(maps, "cilium_throttle")
```

Comment:
s/"cilium_throttle"/bwmap.MapName/?

Reply:
Not a single map in this list uses the package name. Probably begs for some refactoring.